### PR TITLE
Adds a function to return reaced step goal each day for a week

### PR DESCRIPTION
**Pull request by butlertree**
# Description
Added a function to return the step goal reached for each day in a week.  This update includes tests as well. 

# Contributors
@butlertree 

# Checklist
- [x] My PR has an appropriately descriptive and concise title.
- [x] My PR denotes any/all team members who contributed to it.
- [x] My code follows the Turing Style Guides and best practices.
- [x] I ran the code locally and verified that there are no visible errors.
- [x] feat: My PR clearly describes what feature I'm adding and any changes needed to make it work. _(if applicable)_

### CODE REVIEW

**Review by BrendanTurner1 — APPROVED** [review]
Awesome. I was thinking maybe we can create a function that fetches the latest week instead of hardcoding it into these functions.

File: test/scripts-test.js [hunk 1]
```diff
@@ -341,10 +341,72 @@ describe('numberOfStepsGivenDate', () => {
...
   });
-
+  
   it('should return undefined when activityData is not provided', function () {
     const user = {};
     const result = numberOfStepsGivenDate(user, '2023/03/24');
     expect(result).to.be.undefined;
   });
+});
+
```
Comment (ericahagle) [inline test/scripts-test.js:351]: These tests will also need to be refactored to utilize our testData.js file. But that will be done in another PR.

File: src/scriptDefinitions.js [hunk 2]
```diff
@@ -221,6 +221,27 @@ function checkStepGoal(currentUser) {
   }
 };
 
+function checkStepGoal7Days(currentUser) {
```
Comment (ericahagle) [inline src/scriptDefinitions.js:224]: Group review—function is solving what we need! Great work, @butlertree ! 🌴

**Review by ericahagle — APPROVED** [review]
